composer.json

```diff
@@ -19,7 +19,8 @@
     "drupal/coder": "~8.2.0",
     "phpspec/phpspec": "~2.0",
     "phpunit/phpunit": "~4.0",
-    "mockery/mockery": "0.9.4"
+    "mockery/mockery": "0.9.4",
+    "drush-ops/behat-drush-endpoint": "*"
```

Comment:
Since this is now documented as a separate project, with separate installation instructions, is there any point in including it here?